R/peaks-functions.R

```diff
@@ -514,3 +514,45 @@ joinPeaksGnps <- function(x, y, xPrecursorMz = NA_real_,
     }
     x[keep, , drop = FALSE]
 }
+
+#' Function to keep only the monoisotopic peak for groups of (potential)
+#' isotopologues. The peak with the lowest m/z is considered the monoisotopic
+#' peak for each group of isotopologues.
+#'
+#' @importFrom MetaboCoreUtils isotopologues isotopicSubstitutionMatrix
+#'
+#' @author Nir Shahaf, Johannes Rainer
```

Comment:
May be document the `x` param here with the following (taken from the isotopologues documentation)
```
@param x `matrix` with spectrum data (columns `mz` and `intensity`).
```
and point to `MetaboCoreUtils::isotopologues()` for more details.